examples/pennylane/0_Getting_started/0_Getting_started.ipynb

```diff
@@ -787,7 +786,7 @@
      "output_type": "stream",
      "text": [
       "Note: Charges shown are estimates based on your Amazon Braket simulator and quantum processing unit (QPU) task usage. Estimated charges shown may differ from your actual charges. Estimated charges do not factor in any discounts or credits, and you may experience additional charges based on your use of other services such as Amazon Elastic Compute Cloud (Amazon EC2).\n",
-      "Estimated cost to run this example: 0.386 USD\n"
+      "Estimated cost to run this example: 120.011 USD\n"
```

Comment:
I am concerned about the cost here. Can we probably add a warning, or change the example a bit?

Reply:
Good call, thank you. I've replaced Ankaa3 with SV1 in the code block, but left the Ankaa-3 code there commented out.